rest-api/tools/import_organizations.py

```diff
@@ -139,14 +139,29 @@ def _cleanup_old_entities(self, session, row_list, dry_run):
       hpo_id_list = []
       for hpo in hpos_to_remove:
         old_hpo = self.hpo_dao.get_by_name(hpo)
-        if old_hpo:
-          log_prefix = '(dry run) ' if dry_run else ''
-          logging.info(log_prefix + 'Deleting old HPO no longer in Google sheet: %s', old_hpo.name)
+        if old_hpo and old_hpo.isObsolete != ObsoleteStatus.OBSOLETE:
           hpo_id_list.append(old_hpo.hpoId)
           self.deletion_count += 1
+        elif old_hpo.isObsolete == ObsoleteStatus.OBSOLETE:
```

Comment:
Should be if old_hpo and old_hpo.isObsolete... I think.

Reply:
done